contracts/upgradeable_contracts/amb_erc677_to_erc677/BasicAMBErc677ToErc677.sol

```diff
@@ -62,6 +62,12 @@ contract BasicAMBErc677ToErc677 is
         return mediatorContractOnOtherSide();
     }
 
+    /**
```

Comment:
If it becomes specific for the home side only, does it make sense to move it to the corresponding contract then?

Reply:
It is also used in the stake token mediators pair. If we move the method to the `HomeAMBErc677ToErc677` contract, then we will need to place its duplicate in the `BasicStakeTokenMediator` contract.

Reply:
Got you. Thanks!